library/src/blas3/trtri_trsm.hpp

```diff
@@ -258,7 +265,7 @@ rocblas_status rocblas_trtri_trsm_template(rocblas_handle   handle,
         size_t blockSize            = 128;
         size_t tri_elements_to_zero = num_non_tri_elements(rem);
         size_t numBlocks            = (tri_elements_to_zero + blockSize - 1) / blockSize;
-        hipLaunchKernelGGL(rocblas_trtri_batched_fill,
+        hipLaunchKernelGGL(rocblas_trtri_fill<T>,
```

Comment:
Template argument `<T>` is probably unnecessary, especially if it can be deduced from the actual arguments.

Reply:
Argument `A` will either be a T* or a T**, is there a simple way to retrieve type `T` from this, as I need the type to cast a 0 into `rocblas_trti_fill_lower` from this function?

Reply:
using traits?..